### PR TITLE
Clarify that `pad1` and `pad2` must only be filled with zero-value (0x00) bytes

### PR DESCRIPTION
**Pull request by scouten-adobe**
Responding to PR feedback from @lrosenthol in https://github.com/creator-assertions/identity-assertion/pull/75#discussion_r1527635129.

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -219,7 +219,7 @@ The *<<_identity_assertion,identity assertion>>* contains the following data fie
 * `referenced_assertions` contains a list of assertions that are referenced and signed by this _<<_credential_holder,credential holder>>_
 * `signature` contains the raw byte stream of the _<<_credential_holder,credential holder>>’s_ signature
 * `sig_type` defines the data type of signature
-* `pad1` and `pad2` are zero-filled byte strings used to fill space
+* `pad1` and `pad2` are zero-value (`0x00`) byte strings used to fill space
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:222]:
```suggestion
* `pad1` and `pad2` are zero-filled (binary `\0` values) byte strings used to fill space
```

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -258,8 +258,8 @@ identity = {
   "referenced_assertions": [1* $hashed-uri-map],
   "sig_type": tstr .size (1..max-tstr-length), ; A string identifying the data type of the signature field that follows
   "signature": bstr, ; byte string of the signature
-  "pad1": bstr, ; zero-filled byte string used for filling up space
-  ? "pad2": bstr, ; optional zero-filled byte string used for filling up space
+  "pad1": bstr, ; zero-value (`0x00`) byte string used for filling up space
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:261]:
```suggestion
  "pad1": bstr, ; zero-filled (binary `\0` values) byte string used for filling up space
```

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -258,8 +258,8 @@ identity = {
...
+  ? "pad2": bstr, ; optional zero-value (`0x00`) byte string used for filling up space
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:262]:
```suggestion
  ? "pad2": bstr, ; optional zero-filled (binary `\0` values) byte string used for filling up space
```

**Review by lrosenthol — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -219,7 +219,7 @@ The *<<_identity_assertion,identity assertion>>* contains the following data fie
 * `referenced_assertions` contains a list of assertions that are referenced and signed by this _<<_credential_holder,credential holder>>_
 * `signature` contains the raw byte stream of the _<<_credential_holder,credential holder>>’s_ signature
 * `sig_type` defines the data type of signature
-* `pad1` and `pad2` are zero-value (`0x00`) byte strings used to fill space
+* `pad1` and `pad2` are zero-filled (binary `\0` values) byte strings used to fill space
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:222]:
```suggestion
* `pad1` and `pad2` are zero-filled (binary `0x00` values) byte strings used to fill space
```

**Review by lrosenthol — APPROVED** [review]

File: docs/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -219,7 +219,7 @@ The *<<_identity_assertion,identity assertion>>* contains the following data fie
 * `referenced_assertions` contains a list of assertions that are referenced and signed by this _<<_credential_holder,credential holder>>_
 * `signature` contains the raw byte stream of the _<<_credential_holder,credential holder>>’s_ signature
 * `sig_type` defines the data type of signature
-* `pad1` and `pad2` are zero-filled byte strings used to fill space
+* `pad1` and `pad2` are zero-filled (binary `0x00` values) byte strings used to fill space
```
Comment (scouten-adobe) [inline docs/modules/ROOT/pages/index.adoc:222]: Suggested reword: ... are byte strings filled with binary `0x00` values ...

**Comment by scouten-adobe** [comment]
Per 25 March 2024, meeting :+1: to merge with wording changes as noted.

**Pull request merged**